agent-lib/src/main/java/org/terracotta/angela/agent/AgentController.java

```diff
@@ -130,14 +127,14 @@ public boolean installTsa(InstanceId instanceId,
         terracottaInstall = tsaInstalls.computeIfAbsent(instanceId, (iid) -> new TerracottaInstall(workingDir, portAllocator));
       } else {
         kitLocation = new File(kitInstallationPath);
-        if (license !=null) {
-          license.writeToFile(kitLocation);
-        }
+        // Note: do not install the license with: license.writeToFile(kitLocation);
```

Comment:
I would/could also make arguments about not doing anything here other than unzipping the kit to try and better duplicate a users behavior.

Reply:
oh yes of course! I have only but this comment relate to the license because the code that was there before was regarding to the license.

But I can change the comment and put the same at all places if this is what you would like instead ?

That's ok for me whatever you prefer  ;-)